src/runtime/runtime.cs

```diff
@@ -1925,25 +1946,24 @@ internal static IntPtr PyMem_Realloc(IntPtr ptr, long size)
 
         internal static void SetNoSiteFlag()
         {
+            if (_PythonDll == "__Internal")
+            {
+                throw new NotSupportedException("SetNoSiteFlag didn't support on static compile");
```

Comment:
Why should this not be supported? The symbol is just in the running DLL, so `loader.Load(_PythonDll)` has to return essentially the result of `dlopen(NULL, ...)`.

Reply:
Yes, it should be supported. Just a mistake according to my wrong tests. I will recover it back.